test/inlining/rlexe.xml

```diff
@@ -107,6 +107,12 @@
       <compile-flags>-maxinterpretcount:1 -maxsimplejitruncount:0</compile-flags>
     </default>
   </test>
+  <test>
+    <default>
+      <files>applyBailoutArgs.js</files>
+      <compile-flags>-maxinterpretcount:1 -maxsimplejitruncount:1</compile-flags>
```

Comment:
These are the default compile flags for the interpreted run, so you don't need to add them here. It's best to let the dynapogo run not see them.